modules/xfem/src/base/XFEM.C

```diff
@@ -893,7 +893,7 @@ XFEM::cutMeshWithEFA()
 
     new_node->set_n_systems(parent_node->n_systems());
     efa_id_to_new_node.insert(std::make_pair(new_node_id,new_node));
-    _console<<"XFEM added new node: "<<new_node->id()+1<<std::endl;
+    _console << "XFEM added new node: " << new_node->id() <<std::endl;
```

Comment:
Take a look at all the others in the file. We don't mind having them here
and there when you really need output for synchronization, just try to keep
them down as much as possible. It can be expensive on a cluster to flush an
output buffer.

On Fri, Feb 26, 2016 at 12:35 PM Ben Spencer <notifications@github.com>
wrote:

> In modules/xfem/src/base/XFEM.C
> <https://github.com/idaholab/moose/pull/6447#discussion_r54290601>:
>
> > @@ -893,7 +893,7 @@ XFEM::cutMeshWithEFA()
> >
> >      new_node->set_n_systems(parent_node->n_systems());
> >      efa_id_to_new_node.insert(std::make_pair(new_node_id,new_node));
> > -    _console<<"XFEM added new node: "<<new_node->id()+1<<std::endl;
> > +    _console << "XFEM added new node: " << new_node->id() <<std::endl;
>
> OK. I'll fix that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6447/files#r54290601>.
>